docs/developer_guide/getting_started.md

```diff
@@ -64,15 +64,15 @@ Monero-javscript implements keys-only wallets in the MoneroWalletKeys class. You
 var keysOnlyWallet = await MoneroWalletKeys.createWallet({networkType: MoneroNetworkType.STAGENET, language: "English"});
 ```
 
-The createWallet method accepts a [MoneroWalletConfig](moneroecosystem.org/monero-javascript/MoneroWalletConfig) argument. MoneroWalletConfig is a generic class for passing wallet attributes to any monero-javascript wallet creation method. Each wallet class can determine how to create the new wallet by evaluating which MoneroWalletConfig fields are present and which are absent. If the MoneroWalletConfig does not specify any identifying attributes such as an address or seed phrase, the wallet class will generate a random wallet address (In previous versions of monero-javascript, this would have been accomplished with the "createWalletRandom" method).
+The createWallet method accepts a [MoneroWalletConfig](moneroecosystem.org/monero-javascript/MoneroWalletConfig) argument. MoneroWalletConfig is a generic class for passing wallet attributes to any monero-javascript wallet creation method. Each wallet constructor can determine how to create the new wallet by evaluating which MoneroWalletConfig fields are present and which are absent. If the MoneroWalletConfig object passed to createWallet() does not specify any identifying attributes (such as an address or seed phrase), the wallet class will generate a random wallet address.
```

Comment:
Add markup to code: "The `createWallet()` method..."

Reply:
__seed phrase__ should probably read __mnemonic seed phrase__